pkg/kubectl/describe.go

```diff
@@ -560,6 +562,25 @@ func describeReplicationController(controller *api.ReplicationController, events
 	})
 }
 
+// DaemonControllerDescriber generates information about a daemon controller
+// and the pods it has created.
+type DaemonControllerDescriber struct {
+	client.Interface
+}
+
+func (d *DaemonControllerDescriber) Describe(namespace, name string) (string, error) {
+	// TODO (Ananya): Implement Describe.
+	return describeDaemonController("I'm not a Marvel, I'm a DC!") //running, waiting, succeeded, failed)
```

Comment:
(Pro tip: stuff like this occasionally makes it out to production, and it can be *extremely* embarrassing. So I recommend avoiding it, even though it's boring and this particular one is relatively benign.)